src/libraries/System.Net.Http/tests/UnitTests/HttpClientTest.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Http.Tests
+{
+    using Configuration = System.Net.Test.Common.Configuration;
+
+    public class HttpClientTest
+    {
+        [Fact]
+        public async Task GetAsync_Succeeds()
```

Comment:
This test name is odd given that the test is checking `GetAsync` is throwing a `NotImplementedException`.

Reply:
I really don't know what I was trying to achieve, I removed it.